docs/usage/configuration-options.md

```diff
@@ -797,6 +797,20 @@ It will be compiled using Handlebars and the regex `groups` result.
 If `extractVersion` cannot be captured with a named capture group in `matchString` then it can be defined manually using this field.
 It will be compiled using Handlebars and the regex `groups` result.
 
+### matchQueries
```

Comment:
can't we reuse `matchStrings`?

Reply:
Better to not have confusion

Reply:
I'm not sure about this. I think I prefer `matchStrings` too, including because these two fields would be mutually exclusive